pkg/cmd/server/start/node_args.go

```diff
@@ -58,6 +58,10 @@ type NodeArgs struct {
 
 	// Bootstrap is true if the node should rely on the server to set initial configuration.
 	Bootstrap bool
+	// BootstrapConfigName is the name of a config map to read node-config.yaml from.
+	BootstrapConfigName string
+	// BootstrapConfigNamespace is the namespace the config map for bootstrap config is expected to load from.
+	BootstrapConfigNamespace string
```

Comment:
Why another configurable namespace?  Sounds like another thing we will end up removing in the future.

Reply:
This is configurable from the node, not from the master.  Nodes are allowed to point to different namespaces.

Reply:
> This is configurable from the node, not from the master. Nodes are allowed to point to different namespaces.

I think the "why" question still stands?  Why not be prescriptive, at least until someone has a compelling subdivision use-case.

Reply:
Because we don't as a general rule lock clients into namespaces.  I don't see a trend away from clients owning their destiny, and I don't believe in being overly restrictive here.

Reply:
Because we don't as a general rule lock clients into namespaces.  I don't see a trend away from clients owning their destiny, and I don't believe in being overly restrictive here.

Reply:
> Because we don't as a general rule lock clients into namespaces. I don't see a trend away from clients owning their destiny, and I don't believe in being overly restrictive here.

summarizing in-person discussion: not configurable in the server, configurable in the client if someone really wants to wire it.